### PR TITLE
plotly 1.12.4

### PR DESCRIPTION
**Pull request by chohner**

### CODE REVIEW

File: recipes/plotly/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - requests
+    - six
+    - pytz
```
Comment (ocefpaf) [inline recipes/plotly/meta.yaml]: Are you sure that `requests`, `six`, `pytz` are not `run` time dependencies? (Maybe only `six` is also `build` time.)

**Comment by chohner** [comment]
I hope me stumbling forward isn't a problem? First time working with CI and not sure how I should test this locally...

**Comment by patricksnape** [comment]
You can test it locally by just running `conda-build recipes/plotly -c conda-forge` - admittedly only for your current platform - but it will sort out errors like not being able to find the source.

**Comment by chohner** [comment]
Theres a relatively complete nose testsuite on [github](https://github.com/plotly/plotly.py) that is not distributed via [pypi](http://pypi.python.org/pypi/plotly/1.12.4) so I only checked for working versioning without all the optional requirements (nose, matplotlib, numpy, scipy, pandas, ipython).

File: recipes/plotly/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    - setuptools
+    - requests
+    - pytz
+    - six
+
+test:
+  imports:
+    - plotly
+  commands:
+    - python -c "import plotly; print(plotly.version.__version__)"
```
Comment (patricksnape) [inline recipes/plotly/meta.yaml]: This is essentially the same as the line above. This is what the `imports` block does (minus printing the version).

File: recipes/plotly/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+test:
+  imports:
+    - plotly
+
+about:
+  home: https://plot.ly/python/
+  license: MIT
+  summary: 'An interactive, browser-based graphing library for Python'
+
+  # The remaining entries in this section are optional, but recommended
```
Comment (ocefpaf) [inline recipes/plotly/meta.yaml]: It is a minor thing but you don't really need this comment. (We try to keep the recipes on conda-forge more-or-less consistent with each other and adding comments only on things that break that consistency.)

**Comment by ocefpaf** [comment]
This is good to go! Thanks @chohner!!

@patricksnape since you did the review do you want to merge it :wink:

**Pull request merged**